recipes_source/recipes/what_is_state_dict.py

```diff
@@ -1,30 +1,31 @@
 """
-What is a state_dict in PyTorch
-===============================
-In PyTorch, the learnable parameters (i.e. weights and biases) of a
-``torch.nn.Module`` model are contained in the model’s parameters
-(accessed with ``model.parameters()``). A ``state_dict`` is simply a
-Python dictionary object that maps each layer to its parameter tensor.
-
-Introduction
+PyTorch에서 state_dict란 무엇인가요?
+======================================
+PyTorch에서 ``torch.nn.Module`` 모델의 학습 가능한
+매개변수(예. 가중치와 편향)들은 모델의 매개변수에 포함되어 있습니다.
+(model.parameters()로 접근합니다)
```

Comment:
```suggestion
(model.parameters()를 사용해 접근할 수 있습니다)
```
더 직관적인 이해를 위해 위의 방식으로 표현하는 건 어떠신지 제안합니다.

Reply:
그럼 문장이 너무 길어지지 않을까영?

Reply:
위에 문장도 괜찮을것같습니다!